WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -299,6 +305,23 @@ private void handleOpenPageIntent(Intent intent) {
         }
     }
 
+    private void launchZendeskMyTickets() {
+        if (isFinishing() || getIntent() == null) {
```

Comment:
If the intent is not used in the method, we shouldn't be checking for it. Although I can't think of one, there might be exceptions to it in which case we should write a comment for why we are checking the intent.

For this specific case, we already check that the intent is not null and the Zendesk's My Tickets should be open before ever calling the `launchZendeskMyTickets` method, so it's unnecessary.

Fwiw, the alternative way that would check the intent and do all of this would be returning a `boolean` for whether the  Zendesk My Tickets will actually open. In that case we'd actually utilize the `getIntent`, so my initial comment would still hold. This was just given as an example, I don't recommend actually doing it, because the current code is definitely better.

Reply:
thanks for the thorough explanation - went ahead and made the change in a97bd7113d